favourite-colour/options.js

```diff
@@ -6,8 +6,16 @@ async function saveOptions(e) {
 }
 
 async function restoreOptions() {
-  let res = await browser.storage.managed.get('colour');
-  document.querySelector("#managed-colour").innerText = res.colour;
+ try {
+   let res = await browser.storage.managed.get('colour');
+   document.querySelector("#managed-colour").innerText = res.colour || "Could not find 'colour' in managed storage.";
+ } catch(error) {
+   if (error.message === "Managed storage manifest not found") {
+     document.querySelector("#managed-colour").innerText = "Managed storage manifest not found. Make sure it's stored in an appropriate location.";
+   } else {
+     document.querySelector("#managed-colour").innerText = `Unexpected managed storage error: ${error.message}`;
+   }
+ }
```

Comment:
Normalize indentation.

```suggestion
  try {
    let res = await browser.storage.managed.get('colour');
    document.querySelector("#managed-colour").innerText = res.colour || "Could not find 'colour' in managed storage.";
  } catch(error) {
    if (error.message === "Managed storage manifest not found") {
      document.querySelector("#managed-colour").innerText = "Managed storage manifest not found. Make sure it's stored in an appropriate location.";
    } else {
      document.querySelector("#managed-colour").innerText = `Unexpected managed storage error: ${error.message}`;
    }
  }
```